src/instructlab/model/backends/vllm.py

```diff
@@ -120,32 +164,56 @@ def run_vllm(
         background (bool):            Whether the stdout and stderr vLLM should be sent to /dev/null (True)
                                       or stay in the foreground(False).
     Returns:
-        vllm_process (subprocess.Popen): process of the vllm server
```

Comment:
Chat template and model family need to be added to the args list in this function annotation.

Reply:
good catch! thanks